src/org/opensolaris/opengrok/configuration/messages/Messages.java

```diff
@@ -18,16 +18,19 @@
  */
 
 /*
- * Copyright (c) 2016, 2017, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2018, Oracle and/or its affiliates. All rights reserved.
```

Comment:
update wiki https://github.com/oracle/opengrok/wiki/Messages#examples if anything changed

Reply:
I will as soon as this is integrated :)